Add Ser-Od to contribution list

serod11 opened the pull request.

Esh07 commented:
Hey @serod11 thank you for submitting PR. 

I appreciate your effort in following the tutorial correctly. I can see that you’ve added your name and removed blank empty line at the end of file. It’s interesting that your pull request is mergeable, but Git considers it as having two changes: +1 addition and -1 deletion 😄.

However, due to the repository's requirements, you’re only supposed to add one new line. Please read [3rd Bullet point in Guidelines](https://github.com/firstcontributions/first-contributions/issues/35892) for a better understanding.

Please reset the PR and only add a new line in the Contributor file. 

## TODO


If you prefer the command line, please follow this: 

1. Open your command line terminal

```bash
 git checkout serod11
```
2. Now reset the changes you made

```bash
git reset --hard 7d708c6
```
> 🚨 Please disable any code formatter extension before making changes to `Contributors.md'  
3. Open `Contributors.md` file and add a new line with your name 

4. Now commit that changes

```bash
git add Contributors.md
```
```bash
git commit -m"add my name in list"
```

5. Now push the changes to GitHub repo
```bash
git push --force
```

Thanks

serod11 commented:
> Hey @serod11 thank you for submitting PR.
> 
> I appreciate your effort in following the tutorial correctly. I can see that you’ve added your name and removed blank empty line at the end of file. It’s interesting that your pull request is mergeable, but Git considers it as having two changes: +1 addition and -1 deletion 😄.
> 
> However, due to the repository's requirements, you’re only supposed to add one new line. Please read [3rd Bullet point in Guidelines](https://github.com/firstcontributions/first-contributions/issues/35892) for a better understanding.
> 
> Please reset the PR and only add a new line in the Contributor file.
> 
> ## TODO
> If you prefer the command line, please follow this:
> 
> 1. Open your command line terminal
> 
> ```shell
>  git checkout serod11
> ```
> 
> 2. Now reset the changes you made
> 
> ```shell
> git reset --hard 7d708c6
> ```
> 
> > 🚨 Please disable any code formatter extension before making changes to `Contributors.md'
> 
> 3. Open `Contributors.md` file and add a new line with your name
> 4. Now commit that changes
> 
> ```shell
> git add Contributors.md
> ```
> 
> ```shell
> git commit -m"add my name in list"
> ```
> 
> 5. Now push the changes to GitHub repo
> 
> ```shell
> git push --force
> ```
> 
> Thanks

Thank you for the review, I've fixed the problem as you suggested.

serod11 committed:
Add Ser-Od to contribution list
